### PR TITLE
Feat: Added Static Login Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
I have added a Static Login page in the Trendy-T E-Commerce website.
Screenshot (304)

### CODE REVIEW

File: src/Pages/Login-Page/login.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  grid-template-rows: 2.5rem minmax(calc(100vh - 7.5rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+}
+.login__main {
+  padding-top: 3rem;
+  display: flex;
+  justify-content: center;
```
Comment (sushil-mod) [inline src/Pages/Login-Page/login.css:13]: your css is also correct , if possible these two property are same as you mentioned in below class (login-form--new-account) you can you that class and remove here

File: src/Pages/Login-Page/login.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  font-size: var(--font-size-small);
+}
+.login-form__btn--forget-password:hover {
+  color: red;
+  background-color: transparent;
+  border-color: red;
+}
+.login-form--new-account {
+  display: flex;
+  justify-content: center;
```
Comment (sushil-mod) [inline src/Pages/Login-Page/login.css:108]: here

**Review by sushil-mod — APPROVED** [review]
just a suggestion , Overall code is good

**Review by JantuDeb — APPROVED** [review]
UI looks good and the codes are also fine

File: src/Pages/Login-Page/login.css (hunk 2 above)
Comment (SaurabhChawda) [reply src/Pages/Login-Page/login.css:108]: Okay,I Will change ThankYou

**Comment by SaurabhChawda** [comment]
> UI looks good and the codes are also fine

ThankYou

File: src/Pages/Login-Page/login.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  align-items:flex-start;
+  grid-area: main;
+  background-color: var(--secondary-bg-color);
+}
+
+/* Login form Stats Here */
+.login {
+  margin: 0.5rem;
```
Comment (anshikaa12) [inline src/Pages/Login-Page/login.css:21]: for <10 px you don't have to use rem

**Review by anshikaa12 — APPROVED** [review]

File: src/Pages/Login-Page/login.css (hunk 3 above)
Comment (SaurabhChawda) [reply src/Pages/Login-Page/login.css:21]: Okay, I will change it, Thankyou

**Pull request merged**